src/blockchain/log-processors/order-filled/update-orders-and-positions.ts

```diff
@@ -0,0 +1,33 @@
+import { Augur } from "augur.js";
+import { parallel } from "async";
+import * as Knex from "knex";
+import { Address, Bytes32, AsyncCallback, ErrorCallback } from "../../../types";
+import { upsertPositionInMarket } from "./upsert-position-in-market";
+import { convertFixedPointToDecimal } from "../../../utils/convert-fixed-point-to-decimal";
+
+interface OrderFilledOnContractData {
+  amount: string;
+  creatorPositionInMarket: Array<string>;
+  fillerPositionInMarket: Array<string>;
+}
+
+export function updateOrdersAndPositions(db: Knex, augur: Augur, trx: Knex.Transaction, marketID: Address, shareToken: Address, orderID: Bytes32, creator: Address, filler: Address, numTicks: string|number, callback: ErrorCallback): void {
+  const shareTokenPayload = { tx: { to: shareToken } };
```

Comment:
It looks like the argument `shareToken` and the `shareTokenPayload` object are not used?

Reply:
Good catch